### ISSUE TITLE
Double-height meter toolbars prohibited

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#4049]
### Bug description

In previous versions of Audacity the user could create a double-height and extended meter toolbar for either meter toolbar by un-docking, resizing and re-docking

In 3.2.2 when re-docked the resized meter toolbar reverts to being single-height and default length - and is thus a regression on 3.2.1 and earlier

### Steps to reproduce

1. Launch 3.2.1
2. Drag either meter toolbar out of the upper tooldock
3. Drag rightwards to increase the horizontal length of the meter toolbar
4. Drag down wards sufficiently so that the slider is shown above the reticules not over them (i.e. double-height)
5. Drag the meter toolbar back to the upper tooldock
6. Observe: The meter toolbar retains its double-height and extended length - see image 1 below
7. Repeat with 3.2.2
8. Observe: the meter toolbar reverts to single-height and default length  -  see image 2 below

_You get the same with 3.2.2 of after step 6 you just close 3.2.1 and launch 3.2.2_

### Expected behavior

Resized re-docked meter toolbars to retain double-height and extended length

**1) Audacity 3.2.1**
![image](https://user-images.githubusercontent.com/50205984/206864604-1918ef58-adab-4152-8eef-71366f1a6977.png)

### Actual behavior
Meter toolbar reverts to single-height and default length

**2) Audacity 3.2.2**
![image](https://user-images.githubusercontent.com/50205984/206864656-6cd94ca2-81b2-4e19-97a4-5c0b1089b740.png)

### Audacity Version
3.2.2

### Operating system
Windows 10 - but assume all OS

### Additional context
This was originally reported on the Forum by user ImReallyFakeRight:
**3.2.2 Double Height Docking not working**
https://forum.audacityteam.org/viewtopic.php?f=46&t=128191&p=460601

**Comment by petersampsonaudacity** [comment]
> I wonder if this is better served being split into two separate issues?

@Ryan-Miller-Audacity  and @dozzzzer 

Probably so ...

Note that #4057 which was closed as a duplicate (tagged "not planned).  So maybe 
a) 4507 should be reopened
b) part 2) can then be trimmed from this thread (for clarity)
c) the two threads an be cross-reference linked

**Comment by LWinterberg** [comment]
to me, the docked double-high bar has P5/wontfix importance and the reset in meter length P3/P4. That docked double-height meters even are possible feels like an easter egg or bug. So yeah, separate issues seems helpful

**Comment by petersampsonaudacity** [comment]
> That docked double-height meters even are possible feels like an easter egg or bug.

Actually I recall that it was "designed in" by James, I remember testing it for him.

But I'm not sure if it was ever documented - so effectively, yes an easter egg (but one "discovered" and complained about by a Forum users - which is how this bug got logged in the first place).

**Comment by Ryan-Miller-Audacity** [comment]
Secondary elements removed from this ticket and will be tracked in #4057

**Comment by Ryan-Miller-Audacity** [comment]
Leaving this as only concerning double height docking no longer working as the outstanding issue and assigning P5 priority. We can review in triaging sessions if this is something that we want to fix or not.
